v6/core/analytics/analytics.go

```diff
@@ -114,13 +118,13 @@ func (ah *AnalyticsHandler) postImpressions(impressions []model.Impression) erro
 	bodyContent := &model.SDKEventBatch{
 		AnalyticsVersion: "1.0.0",
 		SdkKeyId:         ah.deviceProperties.RolloutKey(),
-		Timestamp:        float64(time.Now().Unix()),
+		Timestamp:        float64(time.Now().UnixMilli()),
 		Platform:         properties[consts.PropertyTypePlatform.Name],
 		SDKVersion:       properties[consts.PropertyTypeLibVersion.Name],
 		Events:           impressions,
 	}
 
-	uri := fmt.Sprintf("%s/%s", ah.uriPath, bodyContent.SdkKeyId)
+	uri := fmt.Sprintf("%s/impression/%s", ah.uriPath, bodyContent.SdkKeyId)
```

Comment:
This was sort-of working before, because the `/impression` was included in the default `Analytics` url in `environment.go`. But putting it there is inconsistent with every other SDK, and makes it hard to override with a custom config (because you'd need custom settings that include `/impression` for Go, but leave it out for _every_ other language).